### PR TITLE
Automated styling update from server

### PR DESCRIPTION
**Pull request by nextcloud-command**
Auto-generated update of our locally cached css styling from
 * https://nextcloud.github.io/server/apps/theming/css/default.css
 * https://nextcloud.github.io/server/core/css/server.css
 * https://nextcloud.github.io/server/dist/icons.css

### CODE REVIEW

File: styleguide/assets/server.css [hunk 1]
```diff
@@ -4944,9 +4931,16 @@ kbd {
   border-left: 3px solid var(--color-success);
 }
 
+/* dark theme overrides for Nextcloud 24 and earlier */
+.theme--dark .toastify.dialogs .toast-close {
+  /* close icon style */
+}
+.theme--dark .toastify.dialogs .toast-close.toast-close::before {
+  background-image: url("./close-dark.svg");
```
Comment (skjnldsv) [inline styleguide/assets/server.css:4939]: Well, that shows an issue xD
https://github.com/nextcloud/nextcloud-dialogs/tree/master/styles

it should be encoded, it doesn't exists there :thinking: .

Comment (skjnldsv) [reply styleguide/assets/server.css:4939]: https://github.com/nextcloud/nextcloud-dialogs/pull/691

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Fixing cypress ...

**Comment by raimund-schluessler** [comment]
Is it intended, that the folder icon isn't visible anymore?

**Pull request merged**